scripts/ci/ci_build_dockerhub.sh

```diff
@@ -67,6 +67,26 @@ echo
 # shellcheck source=scripts/ci/_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/_script_init.sh"
 
+DOCKER_IMAGE_SPEC="${DOCKERHUB_USER}/${DOCKERHUB_REPO}:${DOCKER_TAG}"
+
+#docker pull "${DOCKER_IMAGE_SPEC}"
```

Comment:
```suggestion
```

Reply:
aaaargh/// Testing :)

Reply:
Actually it is not needed. I will enable caching in DockerHub and I will let DockerHub handle the pull for me so that I won't have to do it in the script. I will remove it.